demisto_sdk/commands/common/hook_validations/base_validator.py

```diff
@@ -155,26 +143,39 @@ def tests_registered_in_conf_json_file(self, test_playbooks: list) -> bool:
         no_tests_explicitly = any(test for test in test_playbooks if 'no test' in test.lower())
         if no_tests_explicitly:
             return True
+
         conf_json_tests = self._load_conf_file()['tests']
 
         content_item_id = _get_file_id(self.structure_validator.scheme_name, self.current_file)
         file_type = self.structure_validator.scheme_name
-        not_registered_tests = get_not_registered_tests(conf_json_tests, content_item_id, file_type, test_playbooks)
-        if not_registered_tests:
-            file_type = self.structure_validator.scheme_name
-            if file_type == 'integration':
-                missing_test_configurations = json.dumps([
-                    {'integrations': content_item_id, 'playbookID': test} for test in not_registered_tests
-                ], indent=4).strip('[]')
-            else:
-                missing_test_configurations = json.dumps([
-                    {'playbookID': test} for test in not_registered_tests
-                ], indent=4).strip('[]')
-            error_message = \
-                f'The following TestPlaybooks are not registered in {self.CONF_PATH} file.\n' \
-                f'Please add\n{missing_test_configurations}\nto {self.CONF_PATH} path under \'tests\' key.'
-            print_error(error_message)
-            return False
+        # Test playbook case
+        if 'TestPlaybooks' in self.file_path and file_type == 'playbook':
+            is_configured_test = any(test_config for test_config in conf_json_tests if
+                                     find_test_match(test_config, file_type, test_playbook_id=content_item_id))
+            missing_test_configurations = json.dumps([
+                {'playbookID': content_item_id}
+            ], indent=4).strip('[]')
```

Comment:
why are you passing a list if you remove it afterward? to get the tab? seems redundant